grpc-bridge/src/main/java/dev/resteasy/grpc/bridge/generator/ServiceGrpcExtender.java

```diff
@@ -142,12 +144,16 @@ private void classHeader(Scanner scanner, StringBuilder sb, String fileName) {
     }
 
     private void imports(Scanner scanner, StringBuilder sb, String fileName) {
-        sb.append("import com.google.protobuf.Descriptors.FieldDescriptor;" + LS)
+        sb.append("import static dev.resteasy.grpc.bridge.runtime.Constants.ANY;" + LS)
+                .append("import com.google.protobuf.Descriptors.FieldDescriptor;" + LS)
                 .append("import com.google.protobuf.GeneratedMessageV3;" + LS)
                 .append("import com.google.protobuf.Timestamp;" + LS)
+                .append("import dev.resteasy.grpc.bridge.runtime.Utility;" + LS)
```

Comment:
Is this needed? I don't see any usage being added.

Reply:
Ah, ANY used to be in another class, so it was already referenced. Lines 282 and 292. Very reasonable question, though.